virttest/ceph.py

```diff
@@ -24,12 +25,13 @@ class CephError(Exception):
 
 @error_context.context_aware
 def rbd_image_create(ceph_monitor, rbd_pool_name, rbd_image_name,
-                     rbd_image_size, force_create=False,
-                     ceph_conf=None, keyfile=None):
+                     rbd_image_size, force_create=False, ceph_conf=None,
+                     keyfile=None, rbd_image_namespace=None):
```

Comment:
hi @YongxueHong , rbd_namespace might lead to the confusion that the namespace and rbd pool belong to the same level, but in fact, it might be introduced for the reason that divide one rbd pool into several different logical space, like sub-directories, it's between rbd pool and image, also, one image can only have one namespace at most,  so rbd_image_namespace could be OK.

Reply:
hi @YongxueHong , From ceph document, rados namespaces can be used to provide isolation between rados clients within a pool. A namespace is a new level in the identity of an object (pool/object_locator(namespace, key)/object name). 
Hence, namespace is a sub-directory to pool, and effects placement of images. So I think rbd_image_namespace can show the function well.

Reply:
Better to `rbd_namespace` since the `rbd_pool` and `rbd_image`.

```
							                rbd
							                 |
							   pool1       pool2        pool3
                                                             |
                                        namespace1     namespace2    namespace3
                                             |
                           image1 image2 image3 image3
```

Reply:
hi @YongxueHong , as we talked, I guessed you mean rbd_pool_namespace instead of rbd_namespace, right? Or I misunderstood your comments?

Reply:
Hi, 
Thanks @YongxueHong, @zhencliu , I fully understand about your concern of the level confusion of namespace, I suggest to use rbd_namespace_name, it shows namespace is an independent level like pool, and this variant specifically shows its name attribute.
Is it ok to you all?

Reply:
> hi @YongxueHong , as we talked, I guessed you mean rbd_pool_namespace instead of rbd_namespace, right? Or I misunderstood your comments?

I suggest using `rbd_pool_namespace` before then I think the `rbd_namespace` is more fit, since the existed argument  `rbd_pool` and `rbd_image`, but the `rbd_image` which is not defined by `rbd_pool_image`, so I suggest using `rbd_namespace` is to keep uniformity with `rbd_pool_name`, `rbd_image_name` and `rbd_image_size`.

Reply:
> Hi,
> Thanks @YongxueHong, @zhencliu , I fully understand about your concern of the level confusion of namespace, I suggest to use rbd_namespace_name, it shows namespace is an independent level like pool, and this variant specifically shows its name attribute.
> Is it ok to you all?

@zixi-chen Both `rbd_namespace_name` and `rbd_namespace` are ok for me, how about you? @zhencliu 
or might you another better naming.
thanks.

Reply:
It's OK for me, @zixi-chen @YongxueHong